extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -348,6 +358,36 @@ export class DatabaseUI extends DisposableObject {
     }
   };
 
+  private handleSetDefaultTourDatabase = async (
+    progress: ProgressCallback,
+    token: CancellationToken,
+  ): Promise<void> => {
+    try {
+      if (workspace.workspaceFolders === undefined) {
```

Comment:
Might as well check it's non-empty too.
```suggestion
      if (!(workspace.workspaceFolders?.length)) {
```